app/views/prison/dashboards/show.html.erb

```diff
@@ -1,32 +1,32 @@
 <%= render 'navigation', estate: @estate, visits: @requested_visits %>
 
-<h2 class="heading-large">Latest visit requests</h2>
+<h2 class="heading-large"><%= t('.title') %></h2>
 
 <% if @requested_visits.any? %>
 
   <table>
-    <caption class="visuallyhidden">Visit inbox</caption>
+    <caption class="visuallyhidden"><%= t('.title') %></caption>
     <thead>
       <tr>
-        <th>Prisoner No.</th>
-        <th>Prisoner Name</th>
-        <th>Visit Date</th>
+        <th>Prisoner name</th>
+        <th>Prisoner no.</th>
+        <th>Visit date</th>
         <th>Time slot</th>
-        <th>Lead Visitor</th>
+        <th>Lead visitor</th>
```

Comment:
Before I joined the project the convention was established to use the locale files for everything, with the justification that it maintains a clean separation between content and code, and that content writers are comfortable editing yaml. Personally I'm not overly concerned either way, but for the sake of consistency I think we should put everything or nothing in locale files, and unless there's a good reason to change let's keep using locale files.